util/triehash-ethereum/Cargo.toml

```
@@ -6,6 +6,6 @@ description = "Trie-root helpers, ethereum style"
 license = "GPL-3.0"
 
 [dependencies]
-triehash = { version = "0.4.0", features = ["ethereum"] }
-ethereum-types = "0.4"
+triehash = { version = "0.4.0", default-features = false }
```

```suggestion
triehash = "0.5.0"
```

(There are no features in the crate anymore)